labs/RivePlayer/samples/RivePlayer.Samples/RivePlayer.md

```diff
@@ -0,0 +1,197 @@
+                      Value="{x:Bind SliderRangeExample.Value, Mode=OneWay}" />
+</rive:RivePlayer>
+<Slider x:Name="SliderRangeExample"
+        Maximum="100"
+        Minimum="0"
+        Value="0" />
+```
+
+In the example above, we nest the `rive:NumberInput` strictly under the `rive:RivePlayer` markup.
+
+To create a reference to the number input named `numLook` in the state machine of this Rive file, we set the `Target` property to that input name. Additionally, the `Value` property uses the `x:Bind` markup extension to bind the slider's `Value` property to the number value of the state machine input.
+
+When the slider range value is changed, the new number value will be set on the value of the `numLook` state machine input and the state machine will respond accordingly.
+
+### `<rive:TriggerInput>`
+
+High-level class for a state machine instance of a trigger input type. This should be nested under the `Rive:RivePlayer` element in XAML
+
+### Props
+
+- `Target` - (string) name of the correlated input in the state machine
+- `x:Name` - (string) name for this trigger reference
+
+### Usage
+
+One example using Rive `TriggerInput` classes is binding it to click events of button UI Elements elsewhere in XAML. See below for a code snippet of this:
+
+```xml
+<!-- Example.xaml -->  
+<rive:RivePlayer Width="600"
+                 Height="600"
+                 Source="https://public.rive.app/community/runtime-files/2244-4463-animated-login-screen.riv">
+    <rive:TriggerInput Target="trigSuccess"
+                       x:Name="SuccessTriggerInput" />
+</rive:RivePlayer>
+<Button Content="Success">
+    <i:Interaction.Behaviors>
+        <ic:EventTriggerBehavior EventName="Click">
+            <ic:CallMethodAction MethodName="Fire"
+                                 TargetObject="{x:Bind SuccessTrigger}" />
+        </ic:EventTriggerBehavior>
+    </i:Interaction.Behaviors>
+</Button>
+```
+
+In the example above, we nest the `rive:TriggerInput` strictly under the `rive:RivePlayer` markup.
+
+To create a reference to the trigger input named `trigSuccess` in the state machine of this Rive file, we set the `Target` property to that input name. Additionally, the `x:Name` property defines the name for the class so that the button knows what to bind the click event to to "fire" the trigger.
+
+When the button is clicked, it will call into the `SuccessTriggerInput` input class and invoke the `Fire`  method on the `trigSuccess` Rive trigger input where the state machine will respond accordingly.
+
+Additionally in this example, we use the [EventTriggerBehavior](https://github.com/Microsoft/XamlBehaviors/wiki/EventTriggerBehavior) and [CallMethodAction](https://github.com/Microsoft/XamlBehaviors/wiki/CallMethodAction) APIs from the [XAML Behaviors](https://github.com/Microsoft/XamlBehaviors/wiki) package. You can reference this package by adding the following to your `Dependencies.props`:
+
+```xml
+<!-- WinUI 2 / UWP -->
```

Comment:
This and the dependencies.props is an internal thing to Labs, so app developers just need to do one of the packages based on whatever platform they're using.

Reply:
Is the updated wording better?

Reply:
Is the updated wording better?